ports/stm32/boards/MIKROE_QUAIL/board.json

```diff
@@ -0,0 +1,21 @@
+{
+    "deploy": [
+        "../MIKROE_QUAIL/deploy.md"
+    ],
+    "docs": "",
+    "features": [
+        "mikroBUS"
+    ],
+    "id": "MIKROE-QUAIL",
+    "images": [
+            "quail_hand.jpg",
+            "quail_top.jpg",
+            "quail_bot.jpg",
+            "quail_bus.jpg"
```

Comment:
please submit these images to https://github.com/micropython/micropython-media

also, probably only need one image and `quail_top.jpg` looks the most suitable

Reply:
OK, done.